hphp/runtime/base/runtime-option.h

```diff
@@ -606,6 +606,12 @@ struct RuntimeOption {
   F(int64_t, PerfWarningSampleRate, 1)                                  \
   /* Profiling flags */                                                 \
   F(bool, EnableReverseDataMap, false)                                  \
+  /* PPC64 Option: Minimum immediate size to use TOC */                 \
+  F(uint16_t, ppc64minTOCImmSize, 64)                                   \
+  /* PPC64 Option: Dump the number of elements stored in TOC */         \
+  F(bool, ppc64dumpTOCNelements, false)                                 \
+  /* PPC64 Option: Allow lwz and not only ld to be used for TOC read */ \
+  F(bool, ppc64useTOCLwz, false)                                        \
```

Comment:
ppc64dumpTOCNelements sounds like it would make sense as an admin command (though having it as a runtime option for scripts seems reasonable). The other two certainly sound like configuration settings. Please start them with a capital letter to match the rest of the file (e.g. PPC64...)